### DISCUSSION TITLE
Can I download the GeNN formated output spike data from the Multi Area Model simulation?

### DISCUSSION THREAD

**Discussion by russelljjarvis** [genn-team/genn#529] [General]
https://github.com/BrainsOnBoard/procedural_paper

Larger GPU-accelerated brain simulations with procedural connectivity

Is the data hosted anywhere I can access?

**Comment by neworderofjamie** [comment]
Yeah, it's all available at https://doi.org/10.25377/sussex.12912699

**Comment by russelljjarvis** [comment]
Excellent thanks Jamie. One of these downloads is 14.9GB, I assume it has everything.

**Reply by russelljjarvis** [reply]
I wonder if this data set also includes the artificial inputs that were used to drive and stimulate the network?

**Comment by neworderofjamie** [comment]
As I remember those downloads contain the spike trains from 5 repeats of each simulation. The activity is spontaneous so the only inputs are poisson noise which we don't record.
